ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala

```diff
@@ -92,7 +93,13 @@ class SandboxSubmissionService private (
         recordOnLedger(commands).transform {
           case Failure(error) =>
             logger.warn(s"Submission of command ${commands.commandId.unwrap} has failed.", error)
-            Failure(error)
+            // quick work-around to handle back-pressure coming from Akka
+            //TODO: we should encapsulate the back pressure signal in a proper type and make recordOnLedger return a Future[OK \/ BackPressure]
```

Comment:
can you do this as part of this pr? that feels very hacky....

Reply:
yes, I can. I will literally do the same just in a lower layer.